gui/objects_map/settings_names.py

```diff
@@ -66,6 +67,7 @@
 networkSettingsNetworks_Mainnet_Goerli_sensor = {"container": networkSettingsNetworks_Mainnet_Goerli, "objectName": "walletNetworkDelegate_Mainnet_5_sensor", "id": "sensor", "type": "MouseArea", "unnamed": 1, "visible": True}
 networkSettingsNetowrks_Mainnet_Testlabel = {"container": networkSettingsNetworks_Mainnet_Goerli_sensor, "objectName": "testnetLabel_Mainnet", "type": "StatusBaseText", "visible": True}
 settingsWalletAccountDelegate_Status_account = {"container": settingsContentBase_ScrollView, "objectName": "Status account", "type": "WalletAccountDelegate", "visible": True}
+settingsWalletAccountDelegate = {"container": settingsContentBase_ScrollView, "objectName": RegularExpression("*"), "type": "WalletAccountDelegate", "visible": True}
```

Comment:
RegularExpression("*") means that objectName can have any value, so if you delete "objectName": RegularExpression("*") you will have the same result

Reply:
ya, i know, i just wanted to keep it in object map. why - because there is a list of WalletAccountDelegate items , which object names are matching equally to their names. So i still want to access any of those delegates and i need an instance for that, so i need to have it in object map. I can remove this RegularExpression thing if you like, i just found it a bit more informative, that indeed, for this delegate object name could be any (i set it implicitly later in the code though) @StateOf-Vlado Shall i remove it or keep it?